runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/Table/Table.types.ts

```diff
@@ -31,6 +31,7 @@ export interface DataTableProps<T extends object> {
 
 export interface TableRowExpanderProps {
     row: Row;
+    handleClick?: (row: Row) => void;
```

Comment:
export `Row` from Table.types.ts so we rely only on logic from single source. its our facade for react-table

Reply:
fyi you imported Row react-table in another file

Reply:
moved all HistoryTable.tsx types to Table.types

Reply:
done